.github/CHANGELOG.md

```diff
@@ -48,6 +48,24 @@
 
 <h3>Bug fixes</h3>
 
+<h3>Documentation</h3>
+
+* Updated the docstring of `qml.PolyXP` to reference the new location of internal
+  usage. [(#1262)](https://github.com/PennyLaneAI/pennylane/pull/1262)
+
+* Removes occurrences of the deprecated device argument ``analytic`` from the documentation.
+  [(#1261)](https://github.com/PennyLaneAI/pennylane/pull/1261)
```

Comment:
Would it make sense to have this in 0.15.1 too? Just so that the docs are more polished

Reply:
Unfortunately @antalszava, since these weren't tagged as 'Bug', they were missed for the v0.15.1 milestone. Since they are documentation related, though, we can point users towards the 'latest' version until the v0.16 release :)